lib/handlers.py

```diff
@@ -1084,3 +1086,39 @@ def _handle(cls, view):
     @classmethod
     def _brand_status(cls, status):
         return '𝕜𝕚𝕥𝕖: {}'.format(status)
+
+class NotificationsHandler(sublime_plugin.EventListener):
+    """ Listener which checks if a notifcation should be shown when a view is activated. """
+
+    def on_activated_async(self, view):
+        self.__class__._handle(view)
+    
+    @classmethod
+    def _handle(cls, view):
+        if not _is_view_supported(view):
+            return
+
+        fext = _fext_from_view(view)
+        if fext == "":
+            return
+        
+        if beta.should_show_notif(fext):
+            beta.show_notif(fext)
+
+class OnboardingHandler(sublime_plugin.EventListener):
```

Comment:
Having 2 handlers to handle onboarding behavior works now because JS and Go both don't have onboarding files. However, once they do, it seems like this design would cause both the onboarding file to be opened and the help dialog to be shown because both the `NotificationsHandler._handle` and `OnboardingHandler._handle` methods would be called.

It's probably cleaner to just have one handler to take care of all first-time onboarding logic. I would advocate for keeping the `OnboardingHandler`, since the name makes it clear that's what it does.